RollGen.Tests.Integration.IoC/ExplodeTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using NUnit.Framework;
+using RollGen.Expressions;
+using RollGen.PartialRolls;
+using RollGen.Tests.Integration.Common;
+using System;
+using System.Collections.Generic;
+
+namespace RollGen.Tests.Unit
```

Comment:
Why is this test class in the integration test project?

Reply:
I think I needed to link against libraries already referenced there. Also I wasn't quite sure where this test should be put, anyhow.

Reply:
If you can move this into the unit test project, then I can merge this pull request

Reply:
Sure give me a bit